components/director/cmd/tenantfetcher-svc/main.go

```diff
@@ -403,6 +410,27 @@ func regionDetailsToMap(regionDetails []tenantfetcher.RegionDetails) map[string]
 	return regionDetailsMap
 }
 
+func dependenciesConfigToMap(cfg tenantfetcher.HandlerConfig) (map[string][]tenantfetcher.Dependency, error) {
+	secretData, err := configprovider.ReadConfigFile(cfg.TenantDependenciesConfigPath)
+	if err != nil {
+		return nil, errors.Wrapf(err, "while reading tenant service dependencies config file")
+	}
+
+	dependenciesConfig := make(map[string][]tenantfetcher.Dependency)
+	config, err := configprovider.ParseConfigToJSONMap(secretData)
+	if err != nil {
+		return nil, errors.Wrapf(err, "while parsing tenant service dependencies config file")
+	}
+
+	for region, dependencies := range config {
+		for _, dependency := range dependencies.Array() {
+			dependenciesConfig[region] = append(dependenciesConfig[region], tenantfetcher.Dependency{Xsappname: dependency.String()})
```

Comment:
here the it is the whole dependency tenantfetcher.Dependency (not Xsappname only)